chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -35,6 +35,8 @@
 keystone_settings = {}
 if is_rgw && !(node[:ceph][:keystone_instance].nil? || node[:ceph][:keystone_instance].empty?)
   keystone_settings = KeystoneHelper.keystone_settings(node, @cookbook_name)
+  keystone_settings['s3_auth'] = true
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)